src/clj/ataru/kk_application_payment/kk_application_payment.clj

```diff
@@ -212,10 +217,9 @@
 (defn- exemption-in-application?
   [application]
   (let [answers (util/application-answers-by-key application)]
-    ; TODO: should we allow not answering the question at all in the end? Eg. for Finnish nationality.
     (if-let [exemption-answer (exemption-form-field-name answers)]
       (contains? exemption-field-ok-values (:value exemption-answer))
-      (throw (ex-info "Missing exemption answer" {:application application})))))
+      false)))
```

Comment:
Vastaus hakemusmaksusta vapauttavaan kysymykseen saattaa puuttua kokonaan jos hakija on Suomen kansalainen, ei heitetä poikkeusta jos näin käy.